### ISSUE TITLE
Saved scenarios lost when switching ports

### ISSUE THREAD

**Issue by kdtalgo** [CEMPD/ctools#127]
I ran and a couple of scenarios in Charleston and saved them for later. I then switched to a different port. When I clicked back on the saved Charleston scenarios and the outputs were lost. The map center and zoom level were still saved for the scenarios, however. Are the outputs meant to be deleted after switching ports?

**Comment by joellenb** [comment]
Outputs are displayed upon completion of a scenario and the results may be downloaded at that time. C-PORT does not save outputs for later retrieval.

Jo Ellen

**Comment by kdtalgo** [comment]
As a somewhat newer user of the C-PORT GUI it wasn't obvious to me (at least initially) that the outputs from saved scenarios are deleted when switching ports. I see the following message above the pulldown tab where you select the ports: "Selecting a new location will unlock your map data". My suggestion is to reword this to make it more clear that selecting a new port will delete your progress. Or add a popup asking "Selecting a new location will cause ALL current outputs to be lost. Would you like to download the outputs from your saved scenarios?" 

Again, not really a problem, but just a suggestion.
